### PR TITLE
Update [E-WBM] to providers.yml

### PR DESCRIPTION
**Pull request by eWBM-fido**
created a new one #147 
please update file 'providers.yml'

### CODE REVIEW

File: _data/providers.yml [hunk 1]
```diff
@@ -1,79 +1,86 @@
...
+    url: https://yubico.com
+    img: yubico.png
+    u2f: Yes
+    otp: Yes
+    
+  - name: E-WBM
+    url: https://www.e-wbm.com
+    img: ewbm.png
+    u2f: Yes
+    otq: Yes
```
Comment (alex-nitrokey) [inline _data/providers.yml]: typo: otp

**Comment by alex-nitrokey** [comment]
Please fix the typo (see comment) and please insert the entry in the alphabetical order and we are fine to merge.
thanks.

**Comment by alex-nitrokey** [comment]
btw: does it really support OTP? I couldn't find any hint [here](https://www.e-wbm.com/fido_usb.jsp)...

**Comment by eWBM-fido** [comment]
> btw: does it really support OTP? I couldn't find any hint [here](https://www.e-wbm.com/fido_usb.jsp)...

Oops... fixed Otp service!
Thank you so much.

File: _data/providers.yml (hunk 1 above)
Comment (eWBM-fido) [reply _data/providers.yml]: Fixed!
Yes -> No

**Comment by alex-nitrokey** [comment]
Sorry, but the typo is still there (otq instead of otp). Please just delete the line, as every non-existing line is a 'no' by default.
And again, please insert it in the correct alphabetical order (below of DigiFlak).
Thanks.

**Comment by eWBM-fido** [comment]
Update Typo ! And Hardware Status !
Thank you!

**Pull request merged**